src/ballet/ed25519/fd_ed25519_user.c

```diff
@@ -215,6 +218,7 @@ fd_ed25519_verify( uchar const   msg[], /* msg_sz */
   fd_ed25519_point_t Rcmp[1];
   fd_ed25519_point_neg( Aprime, Aprime );
   fd_ed25519_double_scalar_mul_base( Rcmp, k, Aprime, S );
+  // edwards25519_scalarmuldouble( (void *)Rcmp, (void *)k, (void *)Aprime, (void *)S );
```

Comment:
Not sure if this is supposed to stay here or is just a comment for testing

Reply:
oh no, this file shouldn't be here... now my secret project is reveled 🤣